drivers/scheduler/k8s/k8s.go

```diff
@@ -571,6 +591,21 @@ func (k *k8s) destroyCoreObject(spec interface{}, opts map[string]bool, app *spe
 		}
 
 		logrus.Infof("[%v] Destroyed StorkRule: %v", app.Key, obj.Name)
+	} else if obj, ok := spec.(*v1.Pod); ok {
+		if value, ok := opts[scheduler.OptionsWaitForResourceLeakCleanup]; ok && value {
+			if pods, err = k8sOps.GetPods(obj.Namespace); err != nil {
```

Comment:
this should do a get pod on the pod being deleted and append to the pods list. We use that pod list below to wait for cleanup

Reply:
Using GetPodByName to get that pod and adding it to the list of pods and then assigning that list to the interface 
'pods' that will be returned.